app/src/components/commonlogger.js

```diff
@@ -0,0 +1,15 @@
+const config = require('config');
+
+const CommonLogger = require('../common-logging/common-logger');
+const CommonLoggingHttp = require('../common-logging/common-logging-http');
+const CommonLoggingQueue = require('../common-logging/common-logging-queue');
+const CommonLoggingStdout = require('../common-logging/common-logging-stdout');
```

Comment:
For future consideration, we may want to have the common-logging have a top-level index.js which would require the above classes instead. That way, a user only has to import the library instead of having to refer to 4 discrete requires.

Reply:
Yup, if/when we move to npm package, we will have to consider what is the easiest way for a consumer.